@pmmmwh/react-refresh-webpack-plugin" をversionUpすると起動時エラーが出る

@pmmmwh/react-refresh-webpack-plugin": "0.5.3",
から
@pmmmwh/react-refresh-webpack-plugin": "0.5.7",
に変更すると以下エラーが出る

```
Failed to compile.

Module not found: Error: You attempted to import /Users/konishikenji/Dev/Private/test-various-tools/node_modules/react-refresh/runtime.js which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.
ERROR in ./src/App.tsx 1:40-139
Module not found: Error: You attempted to import /Users/konishikenji/Dev/Private/test-various-tools/node_modules/react-refresh/runtime.js which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.

ERROR in ./src/components/molucules/CommonFrame/CommonFrame.tsx 1:40-139
Module not found: Error: You attempted to import /Users/konishikenji/Dev/Private/test-various-tools/node_modules/react-refresh/runtime.js which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.

ERROR in ./src/components/molucules/IdInfo/IdInfo.tsx 1:40-139
Module not found: Error: You attempted to import /Users/konishikenji/Dev/Private/test-various-tools/node_modules/react-refresh/runtime.js which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.

ERROR in ./src/components/molucules/shortcut-list/shortcut-list.tsx 1:40-139
Module not found: Error: You attempted to import /Users/konishikenji/Dev/Private/test-various-tools/node_modules/react-refresh/runtime.js which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.

ERROR in ./src/index.tsx 1:40-139
Module not found: Error: You attempted to import /Users/konishikenji/Dev/Private/test-various-tools/node_modules/react-refresh/runtime.js which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.

ERROR in ./src/reportWebVitals.ts 1:40-139
Module not found: Error: You attempted to import /Users/konishikenji/Dev/Private/test-various-tools/node_modules/react-refresh/runtime.js which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.

webpack compiled with 6 errors

```

--- a/package.json
+++ b/package.json
@@ -6,7 +6,7 @@
   "devDependencies": {
     "@babel/core": "7.18.6",
     "@babel/plugin-proposal-private-property-in-object": "^7.18.6",
-    "@pmmmwh/react-refresh-webpack-plugin": "0.5.7",
+    "@pmmmwh/react-refresh-webpack-plugin": "0.5.3",
     "@svgr/webpack": "6.2.1",
     "@testing-library/jest-dom": "5.16.4",
     "@testing-library/react": "13.3.0",

